azure-pipelines-gcp-integration.yml

```diff
@@ -117,6 +117,3 @@ stages:
               GOOGLE_APPLICATION_CREDENTIALS: $(gcp_authkey.secureFilePath)
               GE_TEST_GCP_PROJECT: $(GE_TEST_GCP_PROJECT)
               GE_TEST_BIGQUERY_DATASET: $(GE_TEST_BIGQUERY_DATASET)
-          # The pip freeze output could be helpful to reproduce performance test results.
-          - publish: pip-freeze.txt
-            artifact: PipFreeze
```

Comment:
`Pipfreeze` has not been used to reproduce test results, and is now giving errors in our pipeline.

```bash
ApplicationInsightsTelemetrySender correlated 2 events with X-TFS-Session 7df3e132-aada-4a43-b781-0a06fe1923db
##[error]Artifact PipFreeze already exists for build 119157.
Finishing: PublishPipelineArtifact
```